### PR TITLE
Redis cache support

### PR DESCRIPTION
**Pull request by Shaolans**
This PR add the maven dependency for redis cache and update the JHipsterProperties to support Redis.
Related to https://github.com/jhipster/generator-jhipster/pull/10057
https://github.com/jhipster/generator-jhipster/issues/9280

### CODE REVIEW

File: jhipster-dependencies/pom.xml [hunk 1]
```diff
@@ -81,6 +81,7 @@
         <testcontainers.version>1.11.3</testcontainers.version>
         <xmemcached.version>2.4.6</xmemcached.version>
         <xmemcached-provider.version>4.1.3</xmemcached-provider.version>
+        <redisson.version>3.11.1</redisson.version>
```
Comment (DanielFran) [inline jhipster-dependencies/pom.xml]: @Shaolans Please move this property after <prometheus-simpleclient.version>, properties should be by default ordered by name

Comment (Shaolans) [reply jhipster-dependencies/pom.xml]: Done thanks !

**Comment by DanielFran** [comment]
@Shaolans you can merge if you think it is ready at least in this project.

**Comment by Shaolans** [comment]
@DanielFran I do not have the right for merging, but I think the PR on the generator should be approved first to ensure everything is okay :+1:

**Pull request merged**